mwpc/mwpc1/R3BMwpc1Cal2Hit.cxx

```diff
@@ -203,8 +214,12 @@ void R3BMwpc1Cal2Hit::S467()
         // Obtain position Y ----
         qdown = fy[padmy - 1];
         qup = fy[padmy + 1];
-        if (qdown > 0 && qup > 0)
-            y = GetPositionY(qmy, padmy, qdown, qup);
+        if (header->GetExpId()==S522 ||  header->GetExpId()==S509){
+         y = GetPositionYS522(qmy, padmy, qdown, qup);
+	    } else
+        if (qdown > 0 && qup > 0){
+         y = GetPositionY(qmy, padmy, qdown, qup);
+	    }
```

Comment:
Here are some indentation problems, which are also suggested by clang-tidy. I'm kind of surprised that our clang-format checking doesn't work here (@klenze @jose-luis-rs).

Complexity level warning in this case will be fixed by my new PR #837.